### PR TITLE
chore(docs): Update code snippets

### PR DESCRIPTION
**Pull request by dnys1**
- Updates code snippets to be consistent with new types/practices
- Refines language

### CODE REVIEW

**Comment by dnys1** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/celest-dev/website/55?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #53** <a href="https://app.graphite.dev/github/pr/celest-dev/website/53?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #54** <a href="https://app.graphite.dev/github/pr/celest-dev/website/54?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #55** <a href="https://app.graphite.dev/github/pr/celest-dev/website/55?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/celest-dev/website/55?utm_source=stack-comment).

File: docs/functions/creating-functions.md [hunk 1]
```diff
@@ -5,51 +5,44 @@ sidebar_position: 3
...
 
-The above code snippet is all you need to define your functions! Go to your console and run the following command.
+That's all you need to define your API! Go to your console and run the following command.
 
 ```shell
 celest start
 ```
 
-This command will spin up a local environment for you and watch for changes to your Celest Functions to test your changes, and it will also code-generate a Celest client for your frontend Flutter app to make it easier to connect to your Celest Functions.
+A local environment will spin up for you and watch for changes to your Celest Functions. A client library is also generated for your which you can import in your Flutter app to automatically connect to your Celest Functions.
```
Comment (abdallahshaban557) [inline docs/functions/creating-functions.md:37]: Maybe just "a client" rather than "client library"?

File: docs/functions/creating-functions.md [hunk 2]
```diff
@@ -65,19 +58,31 @@ class MyApp extends StatelessWidget {
...
-    return FutureBuilder(
-      // Call your Celest Function using the code-generated client
-      // highlight-next-line
-      future: celest.functions.greeting.sayHello('Celest'),
-      builder: (_, snapshot) => switch (snapshot) {
-        AsyncSnapshot(:final data?) => Text(data),
-        _ => const CircularProgressIndicator(),
-      },
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
```
Comment (abdallahshaban557) [inline docs/functions/creating-functions.md:62]: I don't think we should add this in our code sample

File: docs/functions/data-types.md [hunk 3]
```diff
@@ -2,16 +2,17 @@
 sidebar_position: 6
 ---
 
-# Using custom data types
-
-With Celest Functions, you can use any of the standard Dart types available such as `int` or `string`, and you can also use your own custom data types. Handling the transfer and formatting of data from your Flutter app to your backend, which is called serialization, is handled out-of-the-box in most cases. In situations requiring custom serialization, you can write your own logic which we can use instead for serializing the request/response of your Celest Functions.
+# Defining custom data types
 
+With Celest Functions, you can use any of the core Dart types available such as `int`, `String`, and `DateTime`. You can also use your own custom data types. Celest will handle the transfer and formatting of data from your Flutter app to your backend, aka serialization, out-of-the-box in most cases. In situations requiring custom serialization, you can write your own logic which will be used instead.
```
Comment (abdallahshaban557) [inline docs/functions/data-types.md:7]: I'd rather we type out "aka" - also known as

File: docs/get-started.md [hunk 4]
```diff
@@ -32,23 +32,21 @@ The CLI will also create a folder in your project called `celest`, which will in
...
-    │   │── client.dart # Connects your flutter app to the backend
-    │   ├── models.dart # contains all custom data types
-    │   └── exceptions # contains all custom error exceptions
-    ├── test/ # Folder for adding tests to your backend
-    └── config/
-        └── env.dart # Environment variables
+    │   │── client.dart     # Code-generated client for your Flutter app
+    │   ├── models.dart     # Custom API models
+    │   └── exceptions.dart # Custom API exceptions
+    └── test/               # Tests of your backend
```
Comment (abdallahshaban557) [inline docs/get-started.md:41]: Tests for your backend

File: docs/overview.md [hunk 5]
```diff
@@ -7,12 +7,16 @@ description: Learn more about Celest and how it helps you build the backend for
 # Overview
 
 ## What is Celest?
-Celest is a fully managed cloud platform built for Flutter developers that enables you to build your backend in Dart. You can build the use cases you need without needing additional tooling or having to context switch and use different programming languages. Since you write your backend in Dart, Celest also enables you to share logic and code between your Flutter app and your backend.
+
+Celest is a fully managed cloud platform that enables you to build your backend entirely in Dart. Build the use cases you need without any additional tooling or context switching to different programming languages. Writing your backend in Dart, Celest enables you to share logic and code between your Flutter app and your backend.
```
Comment (abdallahshaban557) [inline docs/overview.md:11]: There is nothing in the intro that talks about Flutter developers. We should have that in there.

Celest is a fully managed cloud platform for Flutter developers that enables you to build your backend entirely in Dart. Build the use cases you need without any additional tooling or context switching to different programming languages. Writing your backend in Dart, Celest enables you to share logic and code between your Flutter app and your backend.

File: docs/overview.md [hunk 6]
```diff
@@ -22,7 +26,16 @@ TODO: add the local iterations launch video.
...
 ## What can I build with Celest?
-At a high level, the following are the "Cloud Widgets" that are available for you to build with Celest.
+
+At a high level, the following "Cloud Widgets" are available for you when building with Celest.
+
+:::info
+
+**Note:** Celest is currently in beta and not all of the Cloud Widgets are available yet. We will be rolling out new features and functionality over the coming months.
+If there's a use case you'd like to see which isn't listed here, please let us know by [creating an issue](https://github.com/celest-dev/celest/issues/new/choose) in our GitHub repo
+or chatting with us on [Discord](https://discord.gg/vgWwpsXH).
```
Comment (abdallahshaban557) [inline docs/overview.md:36]: https://discord.gg/KtMAfSDXGQ

**Comment by dnys1** [comment]
### Merge activity

* **Jan 22, 12:52 AM**: @dnys1 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/celest-dev/website/55).

**Pull request merged**